udfs/community/test_cases.yaml

```diff
@@ -293,4 +293,7 @@ percentage_difference:
   - test:
     input: CAST(1.0 AS FLOAT64), CAST(1000000000 AS INT64)
     expected_output: CAST(2.0 AS FLOAT64)
- 
+getbit:
+  - test:
```

Comment:
please add one more test which checks for an expected 0 result like the following:
input: CAST(23 AS INT64), CAST(3 AS INT64)
expected_output: CAST(0 AS INT64)

Reply:
Hey @epishova any chance you can push this final change through and resolve the merge conflict? This is otherwise ready to go!